docs/reference/remap.cue

```diff
@@ -0,0 +1,94 @@
+package metadata
+
+remap: {
+  errors: {
+    ArgumentError: {
+      description: "Raised when the provided input is not a supported type."
+    }
+    ParseError: {
+      description: "Raised when the provided input cannot be parsed."
+    }
+  }
```

Comment:
We currently don't have any differentiation between error types — it's all represented as free-form strings that get exposed in the logs — but I did have it on my radar to bring a bit more structure to errors (with optional error codes) to allow for us to better document and provide support when people receive errors.